input/data/features.yml

```diff
@@ -4,8 +4,8 @@ feedback:
   - active: true
     dashboard:
       label: New Issue
-      url: https://github.com/IHE/supplement-template/issues/new/choose
+      url: https://github.com/IHE/RO/SMRT/issues/new/choose
```

Comment:
Are the paths for canonical and issues correct like this /RO/SMRT, not /RO.SMRT?